public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+      <link rel="stylesheet" href="./styles.css" />
```

Comment:
стоит использовать путь вида:
```suggestion
      <link rel="stylesheet" href="%PUBLIC_URL%/styles.css" />
```